crates/mun_memory/src/gc.rs

```diff
@@ -1,9 +1,11 @@
+mod array;
 mod mark_sweep;
 mod ptr;
 mod root_ptr;
 
 use crate::r#type::Type;
 use std::marker::PhantomData;
+use std::ptr::NonNull;
```

Comment:
I've been thinking about this and I would prefer to leave these types of formatting as is for now until these things become standardized through rustfmt. I don't wait to stray away from default rustfmt because it introduces extra manual work for all contributors that can probably be automated in the future anyway.